{
    "CVE_data_meta": {
        "ASSIGNER": "cve@mitre.org",
        "ID": "CVE-2016-9392",
        "STATE": "PUBLIC",
        "UPDATED": "2020-06-15"
    },
    "cna-container": {
        "affected": {
            "vendors": [
                {
                    "products": [
                        {
                            "versions": [
                                {
                                    "version_value": "n/a"
                                }
                            ]
                        }
                    ],
                    "vendor_name": "n/a"
                }
            ]
        },
        "descriptions": [
            {
                "lang": "eng",
                "value": "The calcstepsizes function in jpc_dec.c in JasPer before 1.900.17 allows remote attackers to cause a denial of service (assertion failure) via a crafted file."
            }
        ],
        "problemtypes": {
            "descriptions": [
                [
                    {
                        "description": "n/a",
                        "lang": "eng",
                        "type": "text"
                    }
                ]
            ]
        },
        "references": [
            {
                "name": "https://blogs.gentoo.org/ago/2016/11/16/jasper-multiple-assertion-failure",
                "refsource": "MISC",
                "url": "https://blogs.gentoo.org/ago/2016/11/16/jasper-multiple-assertion-failure"
            },
            {
                "name": "https://www.oracle.com/technetwork/security-advisory/cpujan2019-5072801.html",
                "refsource": "CONFIRM",
                "url": "https://www.oracle.com/technetwork/security-advisory/cpujan2019-5072801.html"
            },
            {
                "name": "94377",
                "refsource": "BID",
                "url": "http://www.securityfocus.com/bid/94377"
            },
            {
                "name": "https://github.com/mdadams/jasper/commit/f7038068550fba0e41e1d0c355787f1dcd5bf330",
                "refsource": "CONFIRM",
                "url": "https://github.com/mdadams/jasper/commit/f7038068550fba0e41e1d0c355787f1dcd5bf330"
            },
            {
                "name": "RHSA-2017:1208",
                "refsource": "REDHAT",
                "url": "https://access.redhat.com/errata/RHSA-2017:1208"
            },
            {
                "name": "[oss-security] 20161117 Re: jasper: multiple assertion failures",
                "refsource": "MLIST",
                "url": "http://www.openwall.com/lists/oss-security/2016/11/17/1"
            },
            {
                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1396971",
                "refsource": "CONFIRM",
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1396971"
            },
            {
                "name": "USN-3693-1",
                "refsource": "UBUNTU",
                "url": "https://usn.ubuntu.com/3693-1/"
            }
        ]
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "5.0"
}